lib/std/tar.zig

```diff
@@ -255,6 +263,30 @@ pub fn pipeToFileSystem(dir: std.fs.Dir, reader: anytype, options: Options) !voi
                     }
                 }
             },
+            .GNUTYPE_LONGNAME => {
+                assert(std.mem.eql(u8, header.name(), "././@LongLink"));
```

Comment:
Where is this name guaranteed?

Reply:
the name is mentioned here: https://ftp.gnu.org/old-gnu/Manuals/tar-1.12/html_node/tar_117.html
but i guess this check can be removed as the type should be enough to assume the following chunks to be a filename.

I just matched the Globals in the [GNU standard](https://www.gnu.org/software/tar/manual/html_node/Standard.html). By "consistent", you mean it should be lowercased, without "GNUTYPE_", or what do you propose? 
I thought it may be useful to be able to distinguish which standards the different values come from.

Reply:
At the very least it should follow the style guide and be lowercase.